.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- Provide a general summary of your changes in the Title above -->
+
+### Description
+<!--
+Describe your changes in detail.
+For contribution process guide, look into CONTRIBUTING.md
+
+If this is a relatively large or complex change:
+ - BEFORE creating a PR, find existing or start a new discussion on Issues tab,
+ - if the discussion is concluded, go ahead with this PR,
+ - perhaps describe what alternatives you considered.
+
+If this PR references or fixes an open issue, please link it here
```

Comment:
The paragraph above states that the issue has to exist, so not if - state that the link has to be added.

Reply:
as I wrote above, that paragraph only referes to significant changes. I don't think we should have an issue for each, even the smallest, change 😉